lib/cli.js

```diff
@@ -370,5 +369,6 @@ Promise.resolve()
   .catch((err /*: { stack: any, code: any }*/) => {
     console.log(err.stack); // eslint-disable-line no-console
     const exitCode = typeof err.code === "number" ? err.code : 1;
+    // eslint-disable-next-line no-process-exit
     process.exit(exitCode);
```

Comment:
For consistency (per two lines above) can we add the ESLint disable comment on this same line please, i.e:
• `process.exit(exitCode); // eslint-disable-line no-process-exit`

p.s. For the most art the code base adds ESLint disable comments on the same line when possible.